doc/classes/File.xml

```diff
@@ -48,7 +49,6 @@
 			</argument>
 			<description>
 				Returns [code]true[/code] if the file exists in the given path.
```

Comment:
I agree with @Xrayez and would not remove the note here. It was added as countless users hit this problem and the documentation for `file_exists` is the most natural place to look for an explanation on why it doesn't work as you think it should.

Here's a slightly edited version:
```suggestion
				Returns [code]true[/code] if the file exists in the given path.
				[b]Note:[/b] Many resources types are imported (e.g. textures or sound files), and their source asset will not be included in the exported game, as only the imported version is used. See [method ResourceLoader.exists] for an alternative approach that takes resource remapping into account.
```